test/CMakeLists.txt

```diff
@@ -90,6 +90,7 @@ target_sources(pf_test PRIVATE
   ${PROFINET_SOURCE_DIR}/src/common/pf_lldp.c
   ${PROFINET_SOURCE_DIR}/src/common/pf_file.c
   ${PROFINET_SOURCE_DIR}/src/common/pf_ppm.c
+  ${PROFINET_SOURCE_DIR}/src/common/pf_ppm_driver_sw.c
```

Comment:
Should pf_cpm_driver_sw be added here also? Are there tests for the new modules?

Reply:
Correct. 
pf_cpm_driver_sw will be added. It showed the tests are not really needing the mocks so it worked anyway.

No new tests has been added. The new modules are partly tested by the existing test suite. 
(Same level of test as previous CPM/PPM)